modules/ballerina-core/src/main/java/org/ballerinalang/model/values/BDataTable.java

```diff
@@ -77,21 +77,21 @@ public long getLong(String columnName) {
         return iterator.getLong(columnName);
     }
 
-    public int getInt(int index) {
-        return iterator.getInt(index);
-    }
+//    public int getInt(int index) {
```

Comment:
Wouldn't it be better to keep the name getInt() but return long? Otherwise, it can cause confusion for the developers. We better keep the names as getInt and getFloat.